src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -62,4 +70,39 @@ public ResponseEntity<Object> createPartnership(
             return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
         }
     }
+
+    @PatchMapping("/{" + URL_PARAM_SUBMISSION_ID + "}")
+    public ResponseEntity<Object> updatePartnership(
+            @RequestAttribute(TRANSACTION_KEY) Transaction transaction,
+            @PathVariable(URL_PARAM_SUBMISSION_ID) String submissionId,
+            @RequestBody Map<String, Object> body,
+            @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId,
+            @RequestHeader(value = ERIC_IDENTITY) String userId
+    ) throws JsonProcessingException {
```

Comment:
do you still need the throws JsonProcessingException ? you should be able to remove it now we are catching it in the method.

Reply:
removed